sdk/identity/Azure.Identity/src/ManagedIdentityClient.cs

```diff
@@ -18,8 +20,10 @@ internal class ManagedIdentityClient
 
         // IMDS constants. Docs for IMDS are available here https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http
         private static readonly Uri s_imdsEndpoint = new Uri("http://169.254.169.254/metadata/identity/oauth2/token");
+        private static readonly IPAddress s_imdsHostIp = IPAddress.Parse("169.254.169.254");
```

Comment:
Should this take into account `MSI_ENDPOINT` variable?

Reply:
No this endpoint is specific to IMDS,  not other managed identity environments which use MSI_ENDPOINT.